service/github/github.go

```diff
@@ -96,21 +88,20 @@ func NewGitHubPullRequest(cli *github.Client, owner, repo string, pr int, sha, l
 		sha:       sha,
 		toolName:  toolName,
 		logWriter: githubutils.NewGitHubActionLogWriter(level),
-		wd:        workDir,
 	}, nil
 }
 
 // Post accepts a comment and holds it. Flush method actually posts comments to
 // GitHub in parallel.
 func (g *PullRequest) Post(_ context.Context, c *reviewdog.Comment) error {
-	c.Result.Diagnostic.GetLocation().Path = filepath.ToSlash(filepath.Join(g.wd,
-		c.Result.Diagnostic.GetLocation().GetPath()))
 	g.muComments.Lock()
 	defer g.muComments.Unlock()
 	g.postComments = append(g.postComments, c)
 	return nil
 }
 
+func (*PullRequest) ShouldPrependGitRelDir() bool { return true }
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported method PullRequest.ShouldPrependGitRelDir should have comment or be unexported
<!-- __reviewdog__:ChA0YjVlMDUzNzQwYzYwNzJhEg9nb2xpbnQtY2lyY2xlY2k= -->